model/role.go

```diff
@@ -140,7 +161,21 @@ func init() {
 		PERMISSION_SYSCONSOLE_READ_USERMANAGEMENT_TEAMS.Id,
 		PERMISSION_SYSCONSOLE_READ_USERMANAGEMENT_CHANNELS.Id,
 		PERMISSION_SYSCONSOLE_READ_USERMANAGEMENT_PERMISSIONS.Id,
-		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_WEB_SERVER.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_DATABASE.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_ELASTICSEARCH.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_FILE_STORAGE.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_IMAGE_PROXY.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_SMTP.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_PUSH_NOTIFICATION_SERVER.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_HIGH_AVAILABILITY.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_RATE_LIMITING.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_LOGGING.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_SESSION_LENGTHS.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_PERFORMANCE_MONITORING.Id,
+		PERMISSION_SYSCONSOLE_READ_ENVIRONMENT_DEVELOPER.Id,
+		PERMISSION_READ_ELASTICSEARCH_POST_INDEXING_JOB.Id,
+		PERMISSION_READ_ELASTICSEARCH_POST_AGGREGATION_JOB.Id,
```

Comment:
You can remove these.
```suggestion
```
Line 171 in this file already gives the ancillary permissions.